src/QsCompiler/CompilationManager/ProjectManager.cs

```diff
@@ -75,6 +75,25 @@ public class ProjectProperties
                 : null)
             ?? TargetCapabilityModule.Top;
 
+        public bool TreatCapabilityErrorsAsWarnings
+        {
+            get
+            {
+                if (!this.BuildProperties.TryGetValue(MSBuildProperties.TreatCapabilityErrorsAsWarnings, out string? errorsAsWarnings)
+                    || errorsAsWarnings is null)
+                {
+                    return false;
+                }
+
+                if (!bool.TryParse(errorsAsWarnings, out bool result))
+                {
+                    return false;
+                }
+
+                return result;
+            }
+        }
+
```

Comment:
TryParse is guaranteed to return `false` when given `null` so the logic can be simplified a bit. Consider:
```suggestion
        public bool TreatCapabilityErrorsAsWarnings =>
			this.BuildProperties.TryGetValue(MSBuildProperties.TreatCapabilityErrorsAsWarnings, out var errorsAsWarnings)
			&& bool.TryParse(errorsAsWarnings, out var result)
			&& result;

```